Mathlib/Tactic/Linter/Common.lean

```diff
@@ -0,0 +1,32 @@
+/-
+Copyright (c) 2024 Moritz Firsching All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa, Moritz Firsching
+-/
+
+import Lean.Elab.Command
```

Comment:
```suggestion
import Lean.DeclarationRange
```
not too important, since it is hard not to import `Lean.Elab.Command`, but this file seems to only need this earlier import.